sql/core/src/test/scala/org/apache/spark/sql/execution/vectorized/ColumnarBatchSuite.scala

```diff
@@ -1308,10 +1308,6 @@ class ColumnarBatchSuite extends SparkFunSuite {
       }
     }
 
-    intercept[java.lang.AssertionError] {
-      batch.getRow(100)
```

Comment:
Please add another test in a follow-up PR.

Reply:
Thanks @gatorsmile , I'll put this in another test once I figure out why it wasn't being hit